src/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -176,10 +289,12 @@ const DiffExprCompute = (props) => {
 };
 
 DiffExprCompute.defaultProps = {
+  diffExprType: null,
 };
 
 DiffExprCompute.propTypes = {
   experimentId: PropTypes.string.isRequired,
+  diffExprType: PropTypes.string,
```

Comment:
this should also be required, otherwise wouldn't the code in here fail?

Reply:
No, see [here](https://github.com/biomage-ltd/ui/pull/65/files/457298551f9708358095cb1c9cf881ddf75a6473#diff-b0755e09efec37c88e6905392bcc561eea89f5633c608c60ad8aac226da7a70bR33).